addons/core/translations/resources/en-us.yaml

```diff
@@ -378,6 +378,7 @@ credential-library:
   types:
     vault: Vault
     static: Static
+    username_password: Username & Password
```

Comment:
I'm not sure if its necessary to define username_password type again here since it's already defined on line 394.